gulpfile.js

```diff
@@ -32,12 +32,17 @@ gulp.task('test', function (cb) {
     make('testLegacy', cb);
 });
 
+gulp.task('layout', function (cb) {
+    make('build', cb) &&
+    make('layout', cb);
```

Comment:
Eric, We were planning to use the aggregated zip as used in the build definition : https://mseng.visualstudio.com/VSOnline/Deployment/_build/definitionEditor?definitionId=4444&_a=simple-process